site/app/views/calendar/CalendarView.php

```diff
@@ -25,6 +25,8 @@ public function showCalendar(CalendarInfo $info, array $courses, bool $in_course
 
         $year = (isset($_GET['year']) && $_GET['year'] != "")  ?  (int) $_GET['year']  : (int) date("Y");
         $month = (isset($_GET['month']) && $_GET['month'] != "") ?  (int) $_GET['month'] : (int) date("n");
+        $day = (isset($_GET['day']) && $_GET['day'] != "") ?  (int) $_GET['day'] : (int) date("j");
```

Comment:
I believe if you change this to `!==` it won't complain about loose comparison and you'll be able to not add to the baseline.

Reply:
I did that so that it doesn't need strict comparison between a string vs number day, just like was previously done with the month and year, but the PR would still likely work without the specific day value.